package.json

```diff
@@ -16,10 +16,11 @@
     "bundle": "browserify src/js/main.js > dist/js/script.js",
     "sync": "node src/index.js & browser-sync start --proxy 'localhost:8080' --files 'src'",
     "src:test": "babel --plugins transform-object-rest-spread src/js/pre-logic.js > src/js/logic.js && browserify src/js/main.js > src/js/script.js && npm run sync",
```

Comment:
It would be nice to have the generated code not in the `src` folder. You could have something like this: https://github.com/uasabi/pacman-mcts/blob/render/package.json#L15